docs/guide/nrn_defaults.rst

```diff
@@ -0,0 +1,124 @@
+.. _nrn_defaults:
+
+The appearance of NEURON's graphical windows is controlled by definitions in the file ``$(NEURONHOME)/lib/nrn.defaults`` (under MSWindows it's ``c:\nrnxx\lib\nrn.def`` where ``xx`` is the version number). This is a plain ASCII file that you can edit with a text editor.
+
+What's $(NEURONHOME)?
+
+Start NEURON, and at the oc> prompt type
+
+.. code::
+    Python
+
+    neuronhome()
+
+On my Linux box, this tells me that $(NEURONHOME) is
+
+.. code::
+    Python
```

Comment:
This is a :file: not code.... but also: it should never be capital Python inside a code